drivers/usb_c/vbus/usbc_vbus_numaker.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_numaker_vbus
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/usb_c/usbc_vbus.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/clock_control_numaker.h>
+#include <zephyr/drivers/reset.h>
+#include <zephyr/drivers/pinctrl.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(vbus_numaker, CONFIG_USBC_LOG_LEVEL);
+
+#include <soc.h>
+#include <NuMicro.h>
+
+#include "../tcpc/ucpd_numaker.h"
+
+/* Implementation notes on NuMaker TCPC/PPC/VBUS
+ *
+ * PPC and VBUS rely on TCPC/UTCPD and are just pseudo. They are completely
+ * implemented in TCPC/UTCPD.
+ */
+
+/**
+ * @brief Immutable device context
+ */
+struct numaker_vbus_config {
+	const struct device *tcpc_dev;
+};
+
+/**
+ * @brief Mutable device context
+ */
+struct numaker_vbus_data {
+	uint32_t dummy;
```

Comment:
hey do you actually need this? is it for not having the struct empty?

Reply:
unneeded, removed it